projects/public-search/src/app/patron-profile/patron-profile-loans/patron-profile-loan/patron-profile-loan.component.spec.ts

```diff
@@ -38,7 +38,7 @@ describe('PatronProfileLoanComponent', () => {
       pid: '1',
       end_date: '2021-03-28 12:00:00',
       extension_count: 2,
-      overdue: true,
+      is_late: true,
```

Comment:
What `is_late` means? Perhaps you can add a small comment. Thanks.

Reply:
Done ;-) thx for the review.